tests/framework/web/XmlResponseFormatterTest.php

```diff
@@ -79,7 +81,7 @@ public function formatArrayDataProvider()
                 'b:c' => 'b:c',
                 'a b c' => 'a b c',
                 'äøñ' => 'äøñ',
-            ], "<response><item>1</item><item>2015-06-18</item><item>b:c</item><item>a b c</item><äøñ>äøñ</äøñ></response>\n"],
+            ], "<response><item>1</item><item>2015-06-18</item><b:c>b:c</b:c><item>a b c</item><äøñ>äøñ</äøñ></response>\n"],
```

Comment:
That changes format significantly. Likely will result in broken apps.

Reply:
@samdark uses `item` tag instead `b:c` that's I'm calling "broken app".

Reply:
This part looks alright.